app/models/nflgame.server.ts

```diff
@@ -5,6 +5,12 @@ export type { NFLGame } from '@prisma/client';
 
 export type GameCreate = Omit<NFLGame, 'id'>;
 
+enum NFLGameStatus {
+  Complete = 'complete',
+  InGame = 'in_game',
+  PreGame = 'pre_game',
+}
+
```

Comment:
It might make sense to use a db native enum via Prisma, but I'm not entirely sure these are all the values it can be and I'd rather leave it flexible.